public/scripts/ssooverride.js

```diff
@@ -37,24 +50,12 @@
 
         avatarEl.attr("src", user.avatar);
         usernameEl.text(user.username);
-        if (user.isAdmin) {
-          adminEl.removeClass("hidden");
-          supermentorEl.addClass("hidden");
-          mentorEl.addClass("hidden");
-        } else if (user.isSuperMentor) {
-          adminEl.addClass("hidden");
-          supermentorEl.removeClass("hidden");
-          mentorEl.addClass("hidden");
-        } else if (user.isMentor) {
-          adminEl.addClass("hidden");
-          supermentorEl.addClass("hidden");
-          mentorEl.removeClass("hidden");
-        } else {
-          adminEl.addClass("hidden");
-          supermentorEl.addClass("hidden");
-          mentorEl.addClass("hidden");
-        }
-        profileEl.attr("href", urlTemplate.parse(profileEl.attr("data-href-template")).expand(user));
+
+        // XXXbramble - until we figure out a permenent UI solution, we're hijacking
+        //              what's already there by hiding any UI that isn't working
+        adminEl.addClass("hidden");
```

Comment:
These could each be done in one line above on 43-45 without declaring a variable. I don't think it's necessary to put them inside their own variables since you've removed them elsewhere.